### ISSUE TITLE
Area near the border not clickable in frameless window

### ISSUE THREAD

**Issue by alitavector** [electron/electron#983]
When adding a button on the top corner, the top half of the button is not clickable and the bottom can be clicked. The frameless mode come with transparency border?

```
document.body.appendChild(btn);
btn.style.position='absolute';
btn.style.left='958px';
btn.style.top='-2px';
```

**Comment by zcbenz** [comment]
Can you provide a complete example? And what's the OS you are using?

**Comment by alitavector** [comment]
This above is the a complete example code with javascript main.js (below) on Windows 7 64bit

```
var app = require('app'); // Module to control application life.
var BrowserWindow = require('browser-window'); // Module to create native browser window.
var dialog = require('dialog');

//require('crash-reporter').start();

var mainWindow = null;

app.on('window-all-closed', function() {
  if (process.platform != 'darwin')
    app.quit();
});

app.on('ready', function() {
  mainWindow = new BrowserWindow({
	title: 'Test',
    width: 1016,
    height: 740,
	resizable: false,
	frame: false,
  });

  mainWindow.loadUrl('file://' + __dirname + '/index.html');
  mainWindow.openDevTools();
  
  mainWindow.on('closed', function() {
    mainWindow = null;
  });
  
});


```

**Comment by zcbenz** [comment]
It seems that click events got eaten by some code when mouse is near the border of window.

**Comment by tommoor** [comment]
@alitavector double check you don't have a webkit-app-region style set on the area, as this will prevent clicks.

**Comment by alitavector** [comment]
No, I don't have that in CSS. What is the default value for
webkit-app-region?

**Comment by lealife** [comment]
On Mac OS X has the problem too!

**Comment by dustinblackman** [comment]
I've run in to the same issue on OSX 10.1. My element has to be atleast 20px from the top of a frameless window before the entire element is clickable.

**Comment by dustinblackman** [comment]
To add, it seems like what's in that area in OSX is actually the original OSX title bar but just hidden under all my elements. This is a screenshot with frameless and transparent on, and my background set to 0.1 opacity (with custom OSX buttons to show the area). 
![screen shot 2015-05-21 at 6 49 23 am](https://cloud.githubusercontent.com/assets/5246169/7747118/d4777708-ff85-11e4-9b03-ed4ef5709cb6.png)

**Comment by dustinblackman** [comment]
Bump. ^_^

**Comment by chrisnojima** [comment]
I get this behavior as well, any ideas?

**Comment by dustinblackman** [comment]
This issue was fixed for me somewhere around 0.31.0 or so.

**Comment by chrisnojima** [comment]
I'm on 0.35.2 electron-prebuilt on os x 10.11.1. just using frame: false, resizable: false. My buttons are basically cut in half mouse-wise. It's easy to tell since i have cursor: pointer on and floating over the 20 pixel area where the header is normally i get the normal cursor but if i go 1 pixel below that its the pointer

**Comment by chrisnojima** [comment]
I made a simple reproduction of the issue here: https://github.com/keybase/electron-quick-start/pull/1

**Comment by chrisnojima** [comment]
@zcbenz ^ thoughts?

**Comment by chrisnojima** [comment]
I believe this may only happen on El Capitan

**Comment by gabriel** [comment]
Yeah I can reproduce (bug exists) on El Capitan but Yosemite is ok

**Comment by lukeapage** [comment]
on windows non-resizable windows get this for the area where normally windows would allow you to resize - e.g. the top 5 pixels of a window where a resizable window gets a resize cursor does not get any mouse events even in non-resizable mode.

**Comment by tonyganch** [comment]
The macOS issue is discussed here: https://github.com/electron/electron/issues/5723#issuecomment-265021674

**Comment by bkniffler** [comment]
Is this supposed to be resolved on windows? I'm on 7.0.0-beta.3 and the top area is unresponsive. Even the inspectors element picker does not respond top elements.
